src/porepy/numerics/mixed_dim/pressure_trace.py

```diff
@@ -0,0 +1,186 @@
+"""
+The module contains the class that implement pressure trace discretization,
+no PDE are associated with this discretization.
+"""
+
+import numpy as np
+import scipy.sparse as sps
+
+import porepy as pp
+
+class PressureTrace(
```

Comment:
I guess this is a function used for the DFN methods to hold the pressure at the interface between several grids? I was just wondering if PressureTrace is the best name since it does not have a function assemble_int_bound_pressure_trace ?

Reply:
and also the name "pressure" is not really the best..